webextensions/resources/group-tab.html

```diff
@@ -124,6 +124,10 @@
     mask-image: url("./icons/folder.svg");
   }
 
+  li.use-default-favicon.group-tab {
```

Comment:
`.group-tab` looks unnecessary.

Reply:
After more research I've realized that we just need to add `break-inside: avoid` to the line 125. I'll do this after merging.